examples/misc/histogram.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <!-- Include CARTO VL JS -->
+  <script src="../../../dist/carto-vl.js"></script>
+  <!-- Include Chart JS -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.2/Chart.min.js"></script>
+  <!-- Include Mapbox GL JS -->
+  <script src="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.js"></script>
+  <!-- Include Mapbox GL CSS -->
+  <link href="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.css" rel="stylesheet" />
+  <!-- Include Custom CSS -->
+  <link rel="stylesheet" type="text/css" href="../style.css">
+</head>
+
+<body>
+  <!-- Map goes here -->
+  <div id="map"></div>
+  <aside class="chart-toolbox">
+    <div class="box">
+      <header>
+        <h1>Histogram</h1>
```

Comment:
This looks more like a **bar-chart**. I think we should add a different title (even rename the file): https://www.forbes.com/sites/naomirobbins/2012/01/04/a-histogram-is-not-a-bar-chart/

Another graphical detail is that histograms usually are presented in horizontal and without a space between the bars